### PR TITLE
Ubuntu 12.04 path discovery

### PR DESCRIPTION
**Pull request by splitice**
As discussed in issue #23

### CODE REVIEW

File: config [hunk 1]
```diff
@@ -19,6 +19,18 @@ else
     ngx_feature_path=
     ngx_feature_libs="-ldrizzle"
     . auto/feature
+
```
Comment (agentzh) [inline config]: Style: please remove the line-trailing tab here.

**Comment by agentzh** [comment]
@splitice Thanks for the patch. Please fix the style issue. Otherwise it looks good :)

**Comment by splitice** [comment]
style fixed

**Comment by agentzh** [comment]
@splitice Committed your patch to master. Thanks!

**Pull request closed without merging**